packages/admin-ui/src/inline/inline.tsx

```diff
@@ -4,15 +4,23 @@ import type {
   VSpaceTokens,
   CSSPropAutocomplete,
 } from '@vtex/admin-ui-core'
+import type * as CSS from 'csstype'
 
 export const Inline = createComponent<'div', InlineProps>((props) => {
-  const { vSpace = '$s', hSpace = '$s', ...htmlProps } = props
+  const {
+    vSpace = '$s',
+    hSpace = '$s',
+    noWrap = false,
+    align = 'start',
+    ...htmlProps
+  } = props
 
   return useElement('div', {
     ...htmlProps,
     baseStyle: {
       display: 'flex',
-      flexWrap: 'wrap',
+      flexWrap: noWrap ? 'nowrap' : 'wrap',
```

Comment:
**Suggestion**: I know `nowrap` is a `flex-wrap` valid value, but trying to give a more explicit to the dev about this prop type, maybe we could change its name to something like: `isNoWrap` or `isWrapped` 🤔

Reply:
I will keep this pattern until we discuss it.